index.js

```diff
@@ -10,12 +11,28 @@ class KoaWebMonetization {
     this.buckets = new Map()
     this.balanceEvents = new EventEmitter()
     this.maxBalance = (opts && opts.maxBalance) || Infinity
+    this.cookieName = (opts && opts.cookieName) || '__monetizer'
+    this.cookieOptions = {
+      httpOnly: false
+    }
+    if (opts && opts.cookieOptions) {
+      this.cookieOptions = Object.assign(opts.cookieOptions, this.cookieOptions)
```

Comment:
This means that the user can't override `httpOnly: false`. Is that intentional? Otherwise we could just say `this.cookieOptions = (opts && opts.cookieOptions) || { httpOnly: false }`